decent-dao/fractal-interface
# Implement Activity Feed UI/UX

## Description

This PR implements displaying all sort of events/transactions on DAO's Dashboard page.
It also contains file structure rearranging for better components structuring

## Notes

This PR also adjusts which data is passed from the transaction to the treasury injector.
This PR will lead to follow-up cleanups.
PR is quite big, but all the parts of this PR are directly attached to the functionality implemented, so breaking it up to smaller pieces might lead to contextual lost for reviewer.

## Issue (if applicable)

Closes #508 

## Testing

1. Open any DAO with bunch of proposals and transactions (for example - http://localhost:3000/#/daos/0x68dB43B90E6284019c0513168b514d103F4e1092/dashboard)
2. Observe transactions and proposals appearing in correct historical order (latest at the top)
3. Optionally - create new proposal on DAO you observing, or send transaction to that DAO's treasury - new transaction and/or proposal should appear there as well without need of reloading the page (for that - you need to keep tab with Dashboard opened)

## Screenshots (if applicable)
<img width="1086" alt="Знімок екрана 2022-09-27 о 21 16 33" src="https://user-images.githubusercontent.com/25801162/192605565-a9241c0f-523f-4c29-a012-b9aed182516d.png">
